solarkraft/src/fetcher/storage.ts

```diff
@@ -68,13 +68,33 @@ export interface ContractCallEntry {
     oldFields: FieldsMap
 }
 
+/**
+ * Serializable fetcher state.
+ */
+export interface FetcherState {
+    /**
+     * For every contract id, store the ledger height,
+     * up to which the transactions were fetched.
+     */
+    heights: OrderedMap<string, number>
```

Comment:
Should we use bigint as the value?

Reply:
Yeah, I also thought about it, but Stellar SDK is using `number` so we are compatible with them: https://stellar.github.io/js-stellar-sdk/OperationCallBuilder.html#forLedger